content/unroll-default-arguments.md

```diff
@@ -0,0 +1,817 @@
+---
+layout: sip
+permalink: /sips/:title.html
+stage: completed
+status: under-review
+title: SIP-61 - Unroll Default Arguments for Binary Compatibility
+---
+
+**By: Li Haoyi**
+
+## History
+
+| Date          | Version            |
+|---------------|--------------------|
+| Feb 14th 2024 | Initial Draft      |
+
+## Summary
+
+This SIP proposes an `@unroll` annotation lets you add additional parameters
+to method `def`s,`class` construtors, or `case class`es, without breaking binary 
+compatibility. `@unroll` works by generating "unrolled" or "telescoping" forwarders:
```

Comment:
Backwards compatible default parameters are definitely a pain point and it would be good if we could address this.